app/models/user.server.ts

```diff
@@ -61,6 +62,51 @@ export async function createUser(
       firstName,
       lastName,
       email,
+      groups,
     },
   });
 }
+
+export async function isStudent(username: PrismaUser['username']) {
+  let result;
+  prisma.user.findUnique({ where: { username } }).then((user) => {
```

Comment:
```ts
const { group } = await prisma.user.findUniqueOrThrow({ where: { username } });
// The group will have -dev in it on staging but not on prod
return /mycustomdomain(-dev)?-students/.test(group);
```

Reply:
Oh, I see what you meant by staging vs prod in that issue.